MH-12797, Explain UI Actions

lkiesow opened the pull request:
The events table contains several quick filters and actions. For
example, clicking on a series will automatically add a filter for that
particular series which can be very useful. However, the functionality
of these actions is explained nowhere and users are left to figure that
out by their own.

This patch adds title tags to all undescribed icons and links giving a
very short explanation about the functionality when users hover over
them.

Example:
![screenshot from 2018-03-28 20-20-14](https://user-images.githubusercontent.com/1008395/38048874-fa25b5d8-32c6-11e8-907b-c1d92508e22b.png)

lkiesow committed:
MH-12797, Explain UI Actions (Events)

The events table contains several quick filters and actions. For
example, clicking on a series will automatically add a filter for that
particular series which can be very useful. However, the functionality
of these actions is explained nowhere and users are left to figure that
out by their own.

This patch adds title tags to all undescribed icons and links giving a
very short explanation about the functionality when users hover over
them.

staubesv commented:
Cool idea! I'm on it ;-)

staubesv reviewed (changes requested):
Hi @lkiesow I like the idea, but for reasons of consistency, tooltips are now missing for the following elements:

Events->Series: Column:Series & Actions
Capture->Location: Column:Name & Actions (don't forget delete which is hidden most often)
Systems->Services: Actions (sanitize)
Organization->Users: Actions
Organization->Groups: Actions
Organization->Access Policies: Actions
Configuration->Themes: Actions

staubesv commented on modules/admin-ui/src/main/resources/public/org/opencastproject/adminui/languages/lang-en_US.json:
```diff
@@ -462,7 +462,19 @@
          "NOCONTENT": "No data available",
-         "PUBLISHED": "Published"
+         "PUBLISHED": "Published",
+         "HOVER": {
```
Maybe "TOOLTIP" would be a better choice

lkiesow replied:
done

lkiesow committed:
MH-12797, Explain UI actions (series, locations, …)

This patch adds further title tags for the series and location tables,
the filters and the global administrative user interface.

lkiesow committed:
MH-12797, Rename HOVER to TOOLTIP

This patch renames the HOVER JSON attributes in the language files to
TOOLTIP.

lkiesow commented:
Hi @staubesv, I've added further title tags for the series and location tables, the filters and the global administrative user interface. I'll leave the other tables to you ;-)
